bin/build-project.ps1

```diff
@@ -1,8 +1,9 @@
 [CmdletBinding()]
```

Comment:
I'm actually not sure how to manually trigger this build so I didn't test this.

The appveyor tests don't trigger a build this way. Maybe we should fix that.